### DISCUSSION TITLE
Keymaping does not work in Genshin-Impact

### DISCUSSION THREAD

**Discussion by Eternal9eu** [PlayCover/PlayCover#115] [Q&A]
Imported keymaping does not work. Nothing happens when I press the option key. I tried to reinstall the playcover and reinstall the game but it did not work as well. I also open and close the keymaping edit, but when I close it, only the mouse can use normally(but also not as same as the imported keymaping). No other keyboard will work. 
_**I went through every step, asked many other people who played the same game and tried their guesses, but it didn't work.**_
plz help me

**Comment by Eternal9eu** [comment]
I tried previous version. Same problem was exist

**Comment by johancedaniel** [comment]
I have the same issue, does anyone have a fix? I can't use the keyboard controls anymore

**Comment by ZhichGaming** [comment]
When you press cmd+k, what does it show?

**Reply by johancedaniel** [reply]
I can see the keys that are mapped, but when I try to press those keys in game after closing the keymap editor, nothing happens

**Reply by ZhichGaming** [reply]
Can you send a short video? Of cmd k and trying to use the keymapping. Did you try restarting your mac?

**Comment by notluquis** [comment]
Same error here, only if I import a keymapping file, because when I create my own keymapping, this works.

**Reply by ZhichGaming** [reply]
Could I get more details as to what works and what does not? Where are you getting the keymapping from? And if creating your own keymapping works, then do that. 


If your keymapping is completely unresponsive, tune down your graphic settings and restart your mac. 

If your keymapping is stuttery, try changing your sensitivity in PlayCover settings by opening PlayCover, right clicking on the app you want to change sensitivity in and change sensitivity. It is also possible to change sensitivity for some games in the app. 

Read this page of the documentation: 
https://docs.playcover.io/keymapping/common_issues

**Reply by notluquis** [reply]
I don't know what I did, but I installed the last nightly build (66), deleted the keymapping, created my own, then deleted the keymapping again, and imported a new keymapping, and it worked.

**Comment by shadowofs** [comment]
same issue here. I can only use the mouse to touch the screen. When I first press cmd+k, the key mapping shows. After I close and save the key mapping configs, everything goes unresponsive, even the cmd+k and the mouse disappears. 

OS version `13.2.1`
PlayCover version `2.0.4`
-------------------------------
Update: I find controller work, so it's kinda solved

**Comment by idler8** [comment]
Resolve after restarting the system

**Comment by tfortamal** [comment]
I am having the same issue, I imported the keymapping, then opened Genshin impact, opened keymapping then closed keymapping, then nothing works with the keyboard. only mouse clicks work.
